include/SFML/Graphics/Shader.hpp

```diff
@@ -99,6 +99,12 @@ class SFML_GRAPHICS_API Shader : GlResource
     ////////////////////////////////////////////////////////////
     ~Shader();
 
+    ////////////////////////////////////////////////////////////
+    /// \brief Defaulted move constructor
+    ///
+    ////////////////////////////////////////////////////////////
+    Shader(Shader &&) noexcept = default;
```

Comment:
We use the `Shader&&` convention (ref/ptr symbol next to type).

Maybe we should agree on a parameter name for move constructors and operators? In some cases we will need to implement it. What about `source`?

Reply:
Our existing precedent is to not name parameters in defaulted or deleted special member functions. Can we just keep doing that instead of figuring out what to name unused params?